docs/faq.md

```diff
@@ -1,6 +1,92 @@
+[Click Here](../../guides/forms/how-to-fill-out-DAA) for a step by step guide on how to fill out the DAA.
+
+If you have incompletely or incorrectly filled out your DAA and would like to upload a revised form, [Click Here](../../guides/forms/how-to-fill-out-DAA#uploading-a-revised-daa) for instructions.
+
+Once you have submitted a correctly filled out DAA and have been granted access to one or more [Data Access Units (DAUs)](#what-is-a-data-access-unit), you can continue checking out files from those DAUs until your access expires. Access is generally granted for 1 year, at which point you must submit an Extension Addendum to continue using the data. [Click Here](../../guides/forms/how-to-fill-out-Extension) for a step-by-step guide on how to fill out the Extension Addendum.
+
+#### What is a Data Access Committee?
+
+A St. Jude Cloud **Data Access Committee (DAC)** is group of St. Jude researchers who oversee access to a particular [Data Access Unit (DAU)](#what-is-a-data-access-unit) and evaluate incoming data requests.
+
+The first time you request access to files in 
+a DAU, it is required that you fill out a [Data Access Agreement (DAA)](#what-is-a-data-access-agreement). Access is granted at the DAU level based on the decision of each DAC upon reviewing the DAA.
+
+!!! example
+    For example, if you make a request asking for all of St. Jude's Acute 
+    Lymphoblastic Leukemia sequencing data, you might be asking for data from 
+    multiple different projects (DAUs) here at St. Jude. For the sake of the example,
+    let's say the data you want is spread across three different DAUs. Once
+    you place a request, your application will be routed to the corresponding
+    three data access committees for approval. Since each DAC is made up of
+    different individuals using different criteria for evaluation, you may or
+    may not be approved for access to all of the files. 
+
+#### What is a Data Access Unit?
+
+A St. Jude Cloud **Data Access Unit (DAU)** is a grouping of data that typically corresponds to a project, study, or dataset generated at the same time at the same institution. Each DAU has its own governing body of researchers, the [Data Access Committee](#what-is-a-data-access-committee), who preside over the data and who may grant or deny access. We currently have the 5 DAUs detailed below.
+
+##### Pediatric Cancer Genome Project (PCGP)
+PCGP is a paired-tumor normal dataset focused on discovering the genetic origins of pediatric cancer.
+The Pediatric Cancer Genome Project is a collaboration between St. Jude Children's Research Hospital and the McDonnell Genome Institute at Washington University School of Medicine that sequenced the genomes of over 600 pediatric cancer patients. 
+
+##### St. Jude Lifetime (SJLIFE)
+SJLIFE is a germline-only dataset focused on studying the long-term adverse outcomes associated with cancer and cancer-related therapy.
```

Comment:
I think this line should be removed. It is all stated more clearly in the next section.